### PR TITLE
add antrea in network addon list

### PR DESCRIPTION
**Pull request by luolanzone**
Antrea project is a k8s network CNI which has been added in https://kubernetes.io/docs/concepts/cluster-administration/networking/#antrea, add it to addon installation page.

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ This page lists some of the available add-ons and links to their respective inst
 ## Networking and Network Policy
 
 * [ACI](https://www.github.com/noironetworks/aci-containers) provides integrated container networking and network security with Cisco ACI.
+* [Antrea](https://github.com/vmware-tanzu/antrea) operates at Layer 3/4 to provide networking and security services for Kubernetes, leveraging Open vSwitch as the networking data plane.
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/addons.md:19]:
```suggestion
* [Antrea](https://antrea.io/) operates at Layer 3/4 to provide networking and security services for Kubernetes, leveraging Open vSwitch as the networking data plane.
```
We prefer to link to docs ahead of source code

**Review by sftim — COMMENTED** [review]
Thanks @luolanzone 

/lgtm

File: content/en/docs/concepts/cluster-administration/addons.md (hunk 1 above)
Comment (luolanzone) [reply content/en/docs/concepts/cluster-administration/addons.md:19]: sure, will update it. thanks!

Comment (luolanzone) [reply content/en/docs/concepts/cluster-administration/addons.md]: Hi @sftim , I just updated it, Could you help to review it again? thanks!

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by luolanzone** [comment]
Hi @daminisatya @kbarnard10, Could you help to review? thanks!

**Pull request merged**